post-processing/clickhouse.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-process-env */
+const fs = require('fs').promises;
+const path = require('path');
+const program = require('commander');
+const ProgressBar = require('progress');
+const chalk = require('chalk').default;
+const Clickhouse = require('../reporters/ClickhouseReporter');
+
+program
+    .description(`Import a crawl into clickhouse, or create a row in the crawl table.
+    
+Examples:
+    # import crawl with default crawlId and no metadata
+    clickhouse.js -d /path/to/crawl
+
+    # Create an entry in the crawl table with ID "mycrawl" and import nothing
+    clickhouse.js -c mycrawl --name "This is a crawl" --region US
+    `)
+    .option('-c, --crawlid <id>', 'Crawl ID')
+    .option('--name <crawlname>', 'Name of the crawl')
+    .option('--region <region>', 'Crawl region code')
+    .option('-d --crawldir <dir>', 'Directory of crawl output to import')
+    .parse(process.argv);
+
+const ch = new Clickhouse();
+const crawlName = program.crawlname;
+const crawlRegion = program.region;
+const crawledPagePath = program.crawldir;
+
+// Must provide at least one option
+if (!crawlName && !crawlRegion && !crawledPagePath && !program.crawlId) {
+    program.outputHelp();
+    process.exit(1);
+}
+
+(async () => {
+    /**
+     * @type {string[]}
+     */
+    let pages = [];
+    if (crawledPagePath) {
+        pages = await fs.readdir(crawledPagePath);
```

Comment:
I'd do some filtering, not sure what will happen when you start reading e.g. screenshots and calling `JSON.parse` on them. There is some simple filtering in that other post-processing file.